lean4-infoview/src/infoview/goals.tsx

```diff
@@ -29,38 +29,53 @@ export function goalsToString(goals: InteractiveGoals): string {
     return goals.goals.map(goalToString).join('\n\n')
 }
 
-export function Goal({pos, goal, reverse}: {pos: DocumentPosition, goal: InteractiveGoal, reverse: boolean}) {
+export interface GoalFilterState {
```

Comment:
Let's put a comment on the interface as to which way the `isFoo` flags go. Does `true` mean show or hide?

Reply:
Done